docs/roman/outlier_detection/outlier_detection.rst

```diff
@@ -0,0 +1,162 @@
+     output image grid, given as a real number between 0 and 1. This specifies
+     the size of the footprint, or "dropsize", of a pixel in units of the input
+     pixel size.
+   * The ``kernel`` specifies the form of the kernel function used to distribute flux onto
+     the separate output images.
+   * The ``weight_type`` indicates the type of weighting image to apply with the bad pixel mask.
+     Available options are ``ivm`` (default) for computing and using an inverse-variance map
+     and ``exptime`` for weighting by the exposure time.
+   * The ``good_bits`` parameter specifies what DQ values from the input exposure
+     should be used when resampling to create the output mosaic.  Any pixel with a
+     DQ value not included in this value (or list of values) will be ignored when
+     resampling.
+   * Resampled images will be written out to disk with suffix `_outlier_i2d` by default.
+   * **If resampling is turned off** through the use of the ``resample_data`` parameter,
+     a copy of the unrectified input images (as a ModelContainer)
+     will be used for subsequent processing.
+
+#. Create a median image from all grouped observation mosaics.
+
+   * The median image is created by combining all grouped mosaic images or
+     non-resampled input data (as planes in a
+     `~romancal.datamodels.container.ModelContainer`) pixel-by-pixel.
+   * The ``nlow`` and ``nhigh`` parameters specify how many low and high values
+     to ignore when computing the median for any given pixel.
+   * The ``maskpt`` parameter sets the percentage of the weight image values to
+     use, and any pixel with a weight below this value gets flagged as "bad" and
+     ignored when resampled.
+   * The ``grow`` parameter sets the width, in pixels, beyond the limit set by
+     the rejection algorithm being used, for additional pixels to be rejected in
+     an image.
+   * The median image is written out to disk as `_<asn_id>_median` by default.
+
+#. By default, the median image is blotted back (inverse of resampling) to
+   match each original input image.
+
+   * Blotted images are written out to disk as `_<asn_id>_blot` by default.
+   * **If resampling is turned off**, the median image is compared directly to
+     each input image.
+
+#. Perform statistical comparison between blotted image and original image to identify outliers.
+
+   * This comparison uses the original input images, the blotted
+     median image, and the derivative of the blotted image to
+     create a cosmic ray mask for each input image.
+   * The derivative of the blotted image gets created using the blotted
+     median image to compute the absolute value of the difference between each pixel and
+     its four surrounding neighbors with the largest value being the recorded derivative.
+   * These derivative images are used to flag cosmic rays
+     and other blemishes, such as satellite trails. Where the difference is larger
```

Comment:
I'm not sure how many satellite trails we'll be getting at L2, maybe moving object trails?